api_server.py

```
@@ -229,19 +250,51 @@ def health():
 def root():
     try:
         now = datetime.now()
```

Ruff (0.15.6)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)